packages/EasyCore/tests/Bridge/Symfony/Stubs/NormalizerInterfaceStub.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Symfony\Stubs;
+
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class NormalizerInterfaceStub implements DenormalizerInterface, NormalizerInterface
```

Comment:
Should be simply `NormalizerStub`, not `NormalizerInterfaceStub`.

Reply:
Done.